### PR TITLE
Adding package for Append

### PR DESCRIPTION
**Pull request by lilyclements**
I believe dplyr is a pre-loaded package, but adding it for consistency and so the user can easily see the package name
@dannyparsons this is ready to merge

### CODE REVIEW

File: instat/dlgAppend.vb [hunk 1]
```diff
@@ -50,7 +50,6 @@ Public Class dlgAppend
         ucrChkIncludeIDColumn.bChangeParameterValue = False
         ucrChkIncludeIDColumn.AddToLinkedControls(ucrLinked:=ucrInputIDColName, objValues:={True}, bNewLinkedAddRemoveParameter:=True, bNewLinkedHideIfParameterMissing:=True)
         ucrInputIDColName.bAddRemoveParameter = False
-        ucrInputIDColName.SetLinkedDisplayControl(lblIDColName)
```
Comment (dannyparsons) [inline instat/dlgAppend.vb]: Did this need to be removed?

**Comment by dannyparsons** [comment]
I agree adding about `dplyr` package name.

File: instat/dlgAppend.vb (hunk 1 above)
Comment (lilyclements) [inline instat/dlgAppend.vb]: Sorry that was an accident, good eye!

**Comment by lilyclements** [comment]
@dannyparsons added the removed line back in, this is ready to merge

**Pull request merged**